traffic_ops/traffic_ops_golang/deliveryservice/letsencryptcert.go

```diff
@@ -125,8 +125,8 @@ func (d *DNSProviderTrafficRouter) CleanUp(domain, token, keyAuth string) error
 			return errors.New("Determining rows affected when deleting dns txt record for fqdn '" + fqdn + "' record '" + value + "': " + err.Error())
 		}
 		if rows == 0 {
-			log.Errorf("Zero rows affected when deleting dns txt record for fqdn '" + fqdn + "' record '" + value + "': " + err.Error())
-			return errors.New("Zero rows affected when deleting dns txt record for fqdn '" + fqdn + "' record '" + value + "': " + err.Error())
+			log.Errorf("Zero rows affected when deleting dns txt record for fqdn '" + fqdn + "' record '" + value)
```

Comment:
If you're using Errorf anyway, why not use formatting parameters instead of string concatenation?

Reply:
Actually, nevermind. I just remembered this is a backport. That comment should've gone on the original, no point in whining about it here.

Reply:
good point! im working on updating this in master anyway for the other acme work so i can get that fixed to use formatting parameters